node/src/Commands.ts

```diff
@@ -1820,10 +1896,23 @@ export function createLPos(
     element: string,
     options?: LPosOptions,
 ): command_request.Command {
-    let args: string[] = [key, element];
+    const args: string[] = [key, element];
 
     if (options) {
-        args = args.concat(options.toArgs());
+        if (options.rank !== undefined) {
```

Comment:
Can't you just use `if (options.rank)` without checking explicitly for undefined? Same for the other usages here.

Reply:
+1

Reply:
Be careful.  The string value `""` is also considered falsey - not relevant here, but could be relevant in certain circumstances.  
https://www.freecodecamp.org/news/falsy-values-in-javascript/